x-pack/test/stack_functional_integration/configs/config.stack_functional_integration_base.js

```diff
@@ -16,10 +16,17 @@ const log = new ToolingLog({
   level: 'info',
   writeTo: process.stdout,
 });
+log.info(`REPO_ROOT = ${REPO_ROOT}`);
 log.info(`WORKSPACE in config file ${process.env.WORKSPACE}`);
-const stateFilePath = process.env.WORKSPACE
-  ? `${process.env.WORKSPACE}/qa/envvars.sh`
-  : `${REPO_ROOT}/../integration-test/qa/envvars.sh`;
+
+const INTEGRATION_TEST_ROOT = process.env.WORKSPACE
+  ? process.env.WORKSPACE
+  : `${REPO_ROOT}/../integration-test`;
```

Comment:
```suggestion
const INTEGRATION_TEST_ROOT = process.env.WORKSPACE || `${REPO_ROOT}/../integration-test`;
```

Reply:
done (and used resolve instead of concatenating the REPO_ROOT)